ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/influxdb.j2

```diff
@@ -0,0 +1,46 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - influxdb #}
+{% if influxdb is arista.avd.defined %}
+
+## Influxdb Streaming
+
+### Influxdb Summary
+
+{%     if influxdb.destinations is arista.avd.defined %}
+| Destination | Database | URL | VRF | Username |
+| ----------- | -------- | --- | --- | -------- |
+{%         for destination in influxdb.destinations | arista.avd.natural_sort('name') %}
+| {{ destination.name | arista.avd.default('-') }} | {{ destination.database | arista.avd.default('-') }} | {{ destination.url | arista.avd.default('-') }} | {{ influxdb.vrf | arista.avd.default('-') }} | {{ destination.username | arista.avd.default('-') }} |
+{%         endfor %}
+{%     endif %}
+{%     if influxdb.source_standard_enabled is arista.avd.defined(false) %}
+
+Standard Group Source Enabled : {{ influxdb.source_standard_enabled }}
```

Comment:
Please print this above the tables just after the influxdb summary heading.

Reply:
Fixed.